docs/build/getting-started/multicall.md

```diff
@@ -0,0 +1,186 @@
+---
+description: 'It allows you to bundle multiple calls to various contracts, for both reading and writing data, into a single transaction.'
```

Comment:
```suggestion
description: 'Multicall3 allows you to bundle multiple calls to various contracts, for both reading and writing data, into a single transaction.'
```

Reply:
The first sentence should not start with it.

Reply:
The first sentence should not start with it.

Reply:
@vivekjain23 any reason why this is not yet fixed? I will commit it 👍